### PR TITLE
[KO] Translate 'Ruby 2.6.0-preview2 Released'

### PR DESCRIPTION
**Pull request by riseshia**
Ref:
- Original: https://github.com/ruby/www.ruby-lang.org/pull/1798
- Some useful fix: https://github.com/ruby/www.ruby-lang.org/pull/1799
- one more fix: https://github.com/ruby/www.ruby-lang.org/pull/1801

Could you review this? @ruby/www-ruby-lang-org-i18n-ko

### CODE REVIEW

File: ko/news/_posts/2018-05-31-ruby-2-6-0-preview2-released.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+title: "루비 2.6.0-preview2 릴리스"
+author: "naruse"
+translator: "shia"
+date: 2018-05-31 00:00:00 +0000
+lang: ko
+---
+
+루비 2.6.0-preview2 릴리스를 알리게 되어 기쁩니다.
+
+루비 2.6.0-preview2는 루비 2.6.0의 두번째 프리뷰입니다.
```
Comment (yous) [inline ko/news/_posts/2018-05-31-ruby-2-6-0-preview2-released.md]: 두번째 → 두 번째

File: ko/news/_posts/2018-05-31-ruby-2-6-0-preview2-released.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+또한 지원 환경을 늘려나갈 계획이며, 그 첫 번째로는 Visual Studio를 지원하는 것입니다.
+
+새로운 루비의 성능을 기대해주세요.
+
+## RubyVM::AST [Experimental]
+
+루비 2.6에는 `RubyVM::AST` 모듈이 도입되었습니다.
+
+이 모듈에은 문자열을 파싱하여 AST(추상구문트리)의 Node를 돌려주는 `parse` 메소드, 파일을 파싱하는 `parse_file` 메소드가 들어있습니다.
+`RubyVM::AST::Node`도 도입되었습니다. 이 클래스의 인스턴스로부터 위치정보나 자식 노드를 얻을 수 있습니다. 이 기능은 실험적으로 포함되었으며, 호환성을 보장하지 않습니다.
```
Comment (yous) [inline ko/news/_posts/2018-05-31-ruby-2-6-0-preview2-released.md]: This line seems to be a separate paragraph.

"호환성을 보장하지 않습니다" is ambiguous. How about "AST 노드의 구조는 호환성을 보장하지 않습니다"?

File: ko/news/_posts/2018-05-31-ruby-2-6-0-preview2-released.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+## 새로운 기능
+
+* `Kernel#yield_self`의 별칭으로 `then` 이 추가되었습니다. [[Feature #14594]](https://bugs.ruby-lang.org/issues/14594)
```
Comment (yous) [inline ko/news/_posts/2018-05-31-ruby-2-6-0-preview2-released.md]: \`then\` 이 → \`then\`이

File: ko/news/_posts/2018-05-31-ruby-2-6-0-preview2-released.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+* `rescue` 가 없는 `else` 가 문법 에러가 됩니다. [EXPERIMENTAL]
```
Comment (yous) [inline ko/news/_posts/2018-05-31-ruby-2-6-0-preview2-released.md]: How about `` `rescue` 없는 `else`가``?

File: ko/news/_posts/2018-05-31-ruby-2-6-0-preview2-released.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+* ASCII 이외의 대문자라도 상수를 정의할 수 있게 됩니다. [[Feature #13770]](https://bugs.ruby-lang.org/issues/13770)
```
Comment (yous) [inline ko/news/_posts/2018-05-31-ruby-2-6-0-preview2-released.md]: I think it's about the starting letter. On Ruby 2.5.1, this doesn't cause an error:

``` ruby
>> ÃA = 1
>> ÃA = 2
```

But this causes an error:
``` ruby
>> AÃ = 1
>> AÃ = 2
(irb):2: warning: already initialized constant AÃ
(irb):1: warning: previous definition of AÃ was here
```

File: ko/news/_posts/2018-05-31-ruby-2-6-0-preview2-released.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+
+* 종료 지정이 없는 범위 연산자. [[Feature #12912]](https://bugs.ruby-lang.org/issues/12912)
+
+종료 지정이 없는 범위 연산자, `(1..)`가 추가됩니다. 이는 끝이 없는 것처럼 취급됩니다. 다음은 전형적인 사용 예시입니다.
+
+    ary[1..]                          # ary[1..-1] 와 동치
```
Comment (yous) [inline ko/news/_posts/2018-05-31-ruby-2-6-0-preview2-released.md]: ary[1..-1] 와 → ary[1..-1]와

File: ko/news/_posts/2018-05-31-ruby-2-6-0-preview2-released.md [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+    (1..).each {|index| ... }         # 1로 시작하는 무한 루프
+    ary.zip(1..) {|elem, index| ... } # ary.each.with_index(1) { ... }
+
+* `Binding#source_location`을 추가했습니다. [[Feature #14230]](https://bugs.ruby-lang.org/issues/14230)
+  * 이 메소드는 `binding`의 소스 코드 상의 위치를 `__FILE__`과 `__LINE__`을 가지는 배열로 돌려줍니다. 지금까지는 `eval("[__FILE__, __LINE__]", binding)`을 사용하여 같은 정보를 획득할 수 있었습니다만, `Kernel#eval`이 `binding`의 소스 코드의 위치를 무시하도록 변경할 예정입니다. [[Bug #4352]](https://bugs.ruby-lang.org/issues/4352) 그러므로 앞으로는 `Kernel#eval`보다는 이 새로운 메소드를 사용해야 합니다.
+
+* `Kernal#system`이 실패했을 경우 `false`를 돌려주는 대신, 에러를 던지도록 하는 `:exception` 옵션을 추가. [[Feature #14386]](https://bugs.ruby-lang.org/issues/14386)
```
Comment (yous) [inline ko/news/_posts/2018-05-31-ruby-2-6-0-preview2-released.md]: 추가 → 추가했습니다

File: ko/news/_posts/2018-05-31-ruby-2-6-0-preview2-released.md (hunk 5 above)
Comment (riseshia) [reply ko/news/_posts/2018-05-31-ruby-2-6-0-preview2-released.md]: you're right. 👍 This is just my mistake.

**Comment by riseshia** [comment]
Thanks!!!

**Pull request merged**